app/main.py

```diff
@@ -19,12 +19,6 @@ def _create_ship(self, start, end):
             for row in (range(start[0], end[0] + 1)):
                 self.decks.append(Deck(start[1], row))
 
```

Comment:
maybe Deck(start[1], row) should be Deck(row, start[1])? it passes tests, but if you print the field, you will see, it's wrong